pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -201,20 +201,24 @@ func newChangeAggregatorProcessor(
 ) (_ execinfra.Processor, retErr error) {
 	// Setup monitoring for this node drain.
 	drainWatcher, drainDone := makeDrainWatcher(flowCtx)
+	memMonitor := execinfra.NewMonitor(ctx, flowCtx.Mon, mon.MakeName("changeagg-mem"))
+	var err error
```

Comment:
We should use the named return value `retErr` in the `defer` instead of declaring a new `err`